docs/examples/ex_kp.rst

```diff
@@ -0,0 +1,97 @@
+.. _exkp:
+
+Load Kp from Multiple Sources
+=============================
+
+Following from the previous F\ :sub:`10.7`\ example, :ref:`exf107`,
+:py:mod:`pysatSpaceWeather` has a routine to combine several sources of Kp over
+different time periods. This may be done using the
+:py:func:`~pysatSpaceWeather.instruments.methods.kp_ap.combine_kp` function.
+
+::
+
+
+   import datetime as dt
+   import matplotlib as mpl
+   import matplotlib.pyplot as plt
+   
+   import pysat
+   import pysatSpaceWeather as py_sw
+
+   kp_his = pysat.Instrument(inst_module=py_sw.instruments.sw_kp,
+                             tag='', update_files=True)
+   kp_rec = pysat.Instrument(inst_module=py_sw.instruments.sw_kp,
+                             tag='recent', update_files=True)
+   kp_for = pysat.Instrument(inst_module=py_sw.instruments.sw_kp,
+                             tag='forecast', update_files=True)
+
+   # Set the time range
+   stime = dt.datetime(1999, 1, 1)
+   etime = kp_for.today()
+
+   # If needed, download the data
+   kp_his.download(start=stime, stop=etime)
+   kp_rec.download(start=stime, stop=etime)
+   kp_for.download(start=etime)
+
+   # Combine the Kp sources for all available times
+   kp = py_sw.instruments.methods.kp_ap.combine_kp(kp_his, kp_rec, kp_for,
+                                                   stime, etime)
+
+   # Check the combined Instrument index
+   print(kp.index[0], kp.index[-1])
+
+
+This yeilds ``1999-01-01 00:00:00 2022-08-23 21:00:00``, where the last date is
+your current date (not the date that I created this example).  Now, we can take
```

Comment:
I'm getting the day before, not the current date, though this may just be a data lag